api/src/test/scala/hmda/api/model/ModelGenerators.scala

```diff
@@ -12,7 +12,7 @@ trait ModelGenerators {
     for {
       status <- Gen.oneOf("OK", "SERVICE_UNAVAILABLE")
       service = "hmda-api"
-      time = Calendar.getInstance().getTime().toString
+      time = Calendar.getInstance().getTime.toString
```

Comment:
Why this change? I find that with `Java` getters it's easier to read if we keep the parens as in the original language / api. Either that, or also change `getInstance()` to `getInstance`